src/typescript/frontend/src/middleware.ts

```diff
@@ -20,7 +20,7 @@ export default async function middleware(request: NextRequest) {
   if (MAINTENANCE_MODE && pathname !== "/maintenance") {
     return NextResponse.redirect(new URL(ROUTES.maintenance, request.url));
   }
-  if (pathname === "/test" || pathname === "/verify_status") {
+  if (pathname === "/test" || pathname === "/verify_status" || pathname === "/dexscreener") {
```

Comment:
```suggestion
  if (pathname === "/test" || pathname === "/verify_status" || pathname === ROUTES.dexscreener) {
```

I think you added it below, right? Although we probably should have added the others in there too... 😅